test/test_herk.cc

```diff
@@ -62,6 +63,11 @@ void test_herk_work(Params& params, bool run)
     if (! run)
         return;
 
+    // Check for common invalid combinations
+    if (is_invalid_parameters( params )) {
+        return;
+    }
+
     slate::Options const opts =  {
         {slate::Option::Lookahead, lookahead},
         {slate::Option::Target, target},
```

Comment:
Is the "fix gemmA on device" comment below out-dated? It also exists in `test_her2k`, but not in `test_syr2k` or `test_syrk`.

Reply:
Future work

Reply:
It seems like it might be.  Although, `gemmA` still doesn't work for multiple GPUs per process.